### ISSUE TITLE
After installing the deb package and rebooting, I have no new gestures.   

### ISSUE THREAD

**Issue by MikeyBeez** [JoseExposito/touchegg#400]
Swiping with multiple fingers has no effect.  The touchegg daemon is running.  The config file is in place, ut there is no new behavior.

**Comment by JoseExposito** [comment]
Hi!

Which version did you installed? Did you updated from a previous version? If so, which version?

Could you restart your daemon and copy & paste the logs, please?

```bash
$ sudo systemctl daemon-reload && sudo systemctl restart touchegg && sudo journalctl -u touchegg -b -f
```

Let me know if your input device is listed using that command and if information related to gestures is printed when running `touchegg`.

Also, add information about your system, please. (distro, desktop environment, etc)

**Comment by tiotrom** [comment]
Same here, on Arch + Gnome. Installed via AUR but nothing works. Here's the output:
```
sudo systemctl daemon-reload && sudo systemctl restart touchegg && sudo journalctl -u touchegg -b -f
Journal file /var/log/journal/79c87ef7955d464f832509f6b7af5e18/system@0005b633b8d50b2b-35c48afe58deca3a.journal~ is truncated, ignoring file.
-- Logs begin at Tue 2020-11-24 17:18:25 CET. --
dic 11 20:12:35 tio-gl753vd touchegg[27541]: For more information please visit:
dic 11 20:12:35 tio-gl753vd touchegg[27541]: https://github.com/JoseExposito/touchegg
dic 11 20:12:35 tio-gl753vd touchegg[27541]: Option                Meaning
dic 11 20:12:35 tio-gl753vd touchegg[27541]:  --daemon        Run Touchégg in daemon mode. This mode starts a service that gathers gestures but executes no actions
dic 11 20:12:35 tio-gl753vd touchegg[27541]:  --client        Connect to an existing Touchégg daemon and execute actions in your desktop
dic 11 20:12:35 tio-gl753vd touchegg[27541]: Without arguments Touchégg starts in client mode
dic 11 20:12:35 tio-gl753vd touchegg[27541]: Starting Touchégg in daemon mode
dic 11 20:12:35 tio-gl753vd touchegg[27541]: Starting daemon server...
dic 11 20:12:35 tio-gl753vd touchegg[27541]: Server started
dic 11 20:12:35 tio-gl753vd touchegg[27541]: A list of detected compatible devices will be displayed below:
dic 11 20:12:37 tio-gl753vd touchegg[27541]: Compatible device detected:
dic 11 20:12:37 tio-gl753vd touchegg[27541]:         Name: ELAN1200:00 04F3:3045 Touchpad
dic 11 20:12:37 tio-gl753vd touchegg[27541]:         Size: 103.226mm x 70.9032mm
dic 11 20:12:37 tio-gl753vd touchegg[27541]:         Calculating threshold and animation_finish_threshold. You can tune this values in your service file
dic 11 20:12:37 tio-gl753vd touchegg[27541]:         threshold: 55.8293
dic 11 20:12:37 tio-gl753vd touchegg[27541]:         animation_finish_threshold: 279.147


```

**Comment by tiotrom** [comment]
Update: hm...so I have to manually start "touchegg" to work. Shouldn't this start automatically? Will this work normally after reboot?

**Comment by JoseExposito** [comment]
Yes, after reboot it'll work. At least the deb/rpm packages add a desktop file in `/etc/xdg/autostart` so the client gets executed on start.

**Comment by tiotrom** [comment]
I see. Is it possible to not require a reboot? Maybe to simply create a desktop file that executes "touchegg". Would be good to have a small GUI to enable/disable it, in case users may want that.

**Comment by tiotrom** [comment]
Another mention, on my system there is no "touchegg.desktop" in /etc/xdg/autostart or anywhere else. Is it suppose to be created after the system reboot? I suppose no.

**Comment by JoseExposito** [comment]
It looks like the AUR pkgbuild file is not installing this file:
https://github.com/JoseExposito/touchegg/blob/master/installation/touchegg.desktop
In `/etc/xdg/autostart/touchegg.desktop`. You can manually create it. 

By the way, it seems like the AUR package is outdated. You can try the `touchegg-git` package.

**Comment by jbouter** [comment]
I've gained co-maintainer rights on the AUR package and updated it. I've also just included installation of the desktop file to `/etc/xdg/autostart/touchegg.desktop`. The fixed version in AUR is `touchegg-2.0.4-2`.

**Comment by JoseExposito** [comment]
Thanks!
